src/main/java/com/thevoxelbox/voxelsniper/util/BlockHelper.java

```diff
@@ -7,34 +7,28 @@
 import org.bukkit.Art;
 import org.bukkit.Chunk;
 import org.bukkit.Location;
-import org.bukkit.Material;
 import org.bukkit.World;
 import org.bukkit.block.Block;
 import org.bukkit.entity.Entity;
 import org.bukkit.entity.EntityType;
 import org.bukkit.entity.Painting;
 import org.bukkit.entity.Player;
+import org.bukkit.util.Vector;
 import org.jetbrains.annotations.NotNull;
 
 /**
  * @author Voxel
  */
 public class BlockHelper {
 
-    private static final double DEFAULT_PLAYER_VIEW_HEIGHT = 1.65;
-    private static final double DEFAULT_LOCATION_VIEW_HEIGHT = 0;
-    private static final double DEFAULT_STEP = 0.2;
+    private static final double DEFAULT_STEP = 0.1;

```

Comment:
Welke impact heeft dit?

Reply:
die eerste 2 gebruiken we niet meer, amdat we nu getEyelocation gebruiken. die step kan terug naar 0.2 denk ik, zat even te proberen of het hem acurater zou maken, maar had weinig tot geen effect